rod_cutting_problem/rod_cutting.c

```diff
@@ -0,0 +1,32 @@
+/*
+*  Given a rod of length n length unit and an array of prices that contains prices of all pieces of size smaller than n. 
+*  Determine the maximum new_pricesue obtainable by cutting up the rod and selling the pieces.
+*  Time Complexity: O(n^2)
+*  Space Complexity: O(n)
+*/
+
+#include <stdio.h>
+#include <string.h>
+
+#define max(a, b) (a > b)? a : b;
```

Comment:
Space is need around only Binary operators I think, still there is no harm adding the space will do it. :smile: